go.mod

```diff
@@ -256,6 +255,7 @@ require (
 	github.com/yagipy/maintidx v1.0.0 // indirect
 	github.com/yeya24/promlinter v0.2.0 // indirect
 	github.com/zondax/hid v0.9.1-0.20220302062450-5552068d2266 // indirect
+	github.com/zondax/ledger-go v0.12.2 // indirect
```

Comment:
The `zondax/ledger-go` seems not well maintained. I think we should avoid importing this package into Gaia